packages/blade/src/components/Dropdown/docs/stories.ts

```diff
@@ -604,6 +604,46 @@ const WithControlledMultiSelectStory = `
   export default App;
 `;
 
+const WithBottomAlignedSelectStory = `
```

Comment:
The Bottom aligned dropdown overlay suppose to go upwards? 

<img width="611" alt="image" src="https://github.com/razorpay/blade/assets/35374649/5d78bcf7-af5f-418d-bba1-67bc63e00b4a">

Reply:
Yes, the playground on local won't work. Check this deployed chromatic [link](https://61c19ee8d3d282003ac1d81c-gwxawxwngs.chromatic.com/?path=/story/components-dropdown-with-select--with-bottom-aligned-select&globals=showInternalComponents:true;measureEnabled:false)